Source/Icebreaker/packages.config

```diff
@@ -14,55 +13,6 @@
   <package id="Microsoft.ApplicationInsights.Web" version="2.9.1" targetFramework="net46" />
   <package id="Microsoft.ApplicationInsights.WindowsServer" version="2.9.1" targetFramework="net46" />
   <package id="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" version="2.9.1" targetFramework="net46" />
-  <package id="Microsoft.AspNet.Mvc" version="5.1.0" targetFramework="net461" />
-  <package id="Microsoft.AspNet.Razor" version="3.1.0" targetFramework="net461" />
-  <package id="Microsoft.AspNet.TelemetryCorrelation" version="1.0.5" targetFramework="net46" />
-  <package id="Microsoft.AspNet.WebApi" version="5.2.6" targetFramework="net461" />
```

Comment:
How is this working?!? The project still uses Web API.
Delete your packages folder and make sure this still works.

Reply:
The Microsoft.AspNet packages came in because of the System.Web.Mvc dependency that is required for the DependencyResolver.

Reply:
@aosolis: 

![image](https://user-images.githubusercontent.com/5623947/64052991-69307780-cb35-11e9-9da4-d9eb36643dd3.png)

This is to also answer the reason why Web API is being used